appData/utils.js

```diff
@@ -2,7 +2,7 @@ var nconf = require('../wrio_nconf.js');
 var MongoClient = require('mongodb')
 	.MongoClient;
 
-var url = 'mongodb://' + nconf.get('mongo:user') + ':' + nconf.get('mongo:password') + '@' + nconf.get('mongo:host') + '/' + nconf.get('mongo:dbname');
+var url = 'mongodb://' + nconf.get('db:user') + ':' + nconf.get('db:password') + '@' + nconf.get('db:host') + '/' + nconf.get('db:dbname');
```

Comment:
@balthazzar Now we are using mongo settings record {} config format in all WRIO apps, so better to keep config pattern uniform for all apps.